examples/dashboard_scope.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+#
+# This example shows some examples of scope you can use for dashboards.
+#
+
+import getopt
+import os
+import sys
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])), '..'))
+from sdcclient import SdcClient
+
+# random token, since we're not going to use the API for real
+sdc_token = "bb8754d5-025d-4b88-a815-358fba58c4be"
+
+#
+# Instantiate the SDC client
+#
+sdclient = SdcClient(sdc_token)
```

Comment:
Why don't we convert that `_convert_scope_string_to_expression` into a `@staticmethod`? We are not using `self` at all at its implementation and we can avoid this line and the above `sdc_token`. That should take 2 minutes ;-)

Moreover, if it is marked as a `_private` method, we shouldn't be showing this example. We could make it `public`.

Reply:
`self`: good point. I wanted to have an easy way to access the function from 2 classes (one base of the other). Your way looks simpler.